src/Logging.jl

```diff
@@ -106,8 +106,8 @@ type Logger
 
     Logger(name::AbstractString, level::LogLevel, output::LogIO, parent::Logger) = new(name, level, [output], parent)
     Logger(name::AbstractString, level::LogLevel, output::LogIO) = (x = new(); x.name = name; x.level=level; x.output=[output]; x.parent=x)
-    Logger(name::AbstractString, level::LogLevel, output::Array{LogOutput,1}, parent::Logger) = new(name, level, output, parent)
-    Logger(name::AbstractString, level::LogLevel, output::Array{LogOutput,1}) = (x = new(); x.name = name; x.level=level; x.output=output; x.parent=x)
+    Logger{T<:LogOutput}(name::AbstractString, level::LogLevel, output::Array{T,1}, parent::Logger) = new(name, level, output, parent)
+    Logger{T<:LogOutput}(name::AbstractString, level::LogLevel, output::Array{T,1}) = (x = new(); x.name = name; x.level=level; x.output=output; x.parent=x)
```

Comment:
It should, since `LogOutput <: LogOutput == true` -- I do use multiple LogOutput types, but I don't know that I use this constructor specifically. I can do a quick test, but I don't see why it wouldn't work (unless I'm missing something?)